content/blog/blog-2020-08-26-kudo-0.16.0.md

```diff
@@ -0,0 +1,84 @@
+---
+date: 2020-08-27
+---
+
+# Announcing KUDO 0.16.0
+
+We are happy to release [KUDO v0.16.0](https://github.com/kudobuilder/kudo/releases/tag/v0.16.0), with focus on upgradability, support for immutable parameters and a lot of other small improvements and bugfixes.
+
+<!-- more -->
+
+To see the full changelog and the list of contributors who contributed to this release, visit [the Github Release](https://github.com/kudobuilder/kudo/releases/tag/v0.16.0) page.
+
+## KUDO Upgrades
+
+Until recently, `kubectl kudo init` would simply apply the expected state to the cluster and ignore any breaking changes that may have been introduced. This is now changed, from now on:
+
+- `kubectl kudo init` will fail if it detects an existing KUDO installation to prevent accidental upgrades.
+- You can use `kubectl kudo init --upgrade` to upgrade an existing installation.
+
+The upgrade process will verify that the upgrade can be executed correctly and - in later versions - will check installed operators and may contain migrations and other features that have to be done in an upgrade. 
+
+In the same vein we started to integrate upgrade-tests to our CI to ensure that new KUDO releases won't break existing installations and can be upgraded with a defined process.
+
+Run `kubectl kudo init --help` for more details.
+
+## Immutable Parameters
+
+Operators sometimes have the need to specify parameters that are effectively immutable. For example the `NUM_TOKENS` parameter in the KUDO Cassandra operator must not be changed in an update to the instance because Cassandra does not allow this property to be changed. Until now it was not really possible to define a parameter as immutable. This has now changed, with KUDO 0.16.0 it is possible to define a parameter like this:
+
+```yaml
+  - name: NUM_TOKENS
+    description: "The number of tokens assigned to each node."
+    default: "256"
+    immutable: true
+```
+
+This will prevent the user from ever changing the parameter value in an update to the instance. For more details about immutable parameters, take a look at [KEP-30](https://github.com/kudobuilder/kudo/blob/main/keps/0030-immutable-parameters.md). The document also defines the details of changing an existing parameter to immutable in a new operator version.
+
+
+## New Repository Index
+
+The default repository when installing operators with KUDO is the community repository. Adding new operators to this repository has been greatly simplified with a new index of operators and a new community repository URL that is used starting with KUDO 0.16.
```

Comment:
IF we changed to a new community repo... we definitely need a big warning and highlight an action plan for users

Reply:
Very good point! I've added a section on that with it's own anchor.